pkg/util/ssh.go

```diff
@@ -199,10 +200,15 @@ func runSSHCommand(dialer sshDialer, cmd, user, host string, signer ssh.Signer)
 		} else {
 			// Some other kind of error happened (e.g. an IOError); consider the
 			// SSH unsuccessful.
-			err = fmt.Errorf("failed running `%s` on %s@%s: '%v'", cmd, user, host, err)
+			err = fmt.Errorf("failed running `%s` on %s: '%v'", cmd, remote, err)
 		}
 	}
-	return bout.String(), berr.String(), code, err
+	stdout, stderr := bout.String(), berr.String()
+	glog.Infof("[%s] stdout:    %q", remote, stdout)
+	glog.Infof("[%s] stderr:    %q", remote, stderr)
+	glog.Infof("[%s] exit code: %d", remote, code)
+	glog.Infof("[%s] error:     %v", remote, err)
```

Comment:
The err should be `nil` on success, so I don't think I should put this there (Go idiom as far as I understand).

Is V(4) displayed by default?

It would be too bad if we copied and pasted this code to all callers. I could move it one level up, into the e2e util itself, if you're worried about someone else calling this and not wanting it?